analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/InitializationVectorShouldBeRandomTest.cs

```diff
@@ -46,27 +46,18 @@ public class InitializationVectorShouldBeRandomTest
                 .WithOptions(ParseOptionsHelper.FromCSharp8)
                 .Verify();
 
-        [Ignore] // ToDo: Remove after S3329 implementation
         [TestMethod]
         public void InitializationVectorShouldBeRandom_Roslyn_CS() =>
             roslynCS.AddPaths("InitializationVectorShouldBeRandom.cs")
             .Verify();
 
-        [Ignore] // ToDo: Remove after S3329 implementation
-        [TestMethod]
-        public void InitializationVectorShouldBeRandom_Roslyn_CSharp8() =>
-            roslynCS.AddPaths("InitializationVectorShouldBeRandom.CSharp8.cs")
-                .WithOptions(ParseOptionsHelper.FromCSharp8)
-                .Verify();
-
```

Comment:
Why did you move this to `#NET`?

Reply:
Some mess from rebase or something similar.
I moved back where it was.